assets/js/modules/analytics-4/datastore/audience-settings.test.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * `modules/analytics-4` data store: audience settings tests.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { createRegistry } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import { untilResolved } from '../../../../../tests/js/utils';
+import { MODULES_ANALYTICS_4 } from './constants';
+import { availableAudiences as availableAudiencesFixture } from './__fixtures__';
+import { CORE_USER } from '../../../googlesitekit/datastore/user/constants';
+
+/**
+ * To-Do: After the implementation of #8888, we can remove these individual imports
+ * and directly use createTestRegistry from 'tests/js/utils' so that all the stores
+ * are registered without the duplicate registration issue.
+ */
+import AudienceSettingsStore from './audience-settings';
+import AudiencesStore from './audiences';
+import UserAudienceSettingsStore from '../../../googlesitekit/datastore/user/audience-settings';
+import AuthenticationStore from '../../../googlesitekit/datastore/user/authentication';
+import { combineStores, commonStore } from 'googlesitekit-data';
+
+describe( 'modules/analytics-4 audience settings', () => {
+	let registry;
+
+	const saveAudienceSettingsEndpoint = new RegExp(
+		'^/google-site-kit/v1/modules/analytics-4/data/save-audience-settings'
+	);
+
+	const syncAvailableAudiencesEndpoint = new RegExp(
+		'^/google-site-kit/v1/modules/analytics-4/data/sync-audiences'
+	);
+
+	beforeEach( () => {
+		// To-Do: After #8888 is implemented, we can directly use createTestRegistry from 'tests/js/utils'
```

Comment:
As above, plus include a bit of additional detail and a full stop:

```suggestion
		// TODO: After #8888 is implemented, we can directly use `createTestRegistry` from 'tests/js/utils',
		// rather than registering the stores below.
```